encode.py

```diff
@@ -2,18 +2,27 @@
 import torch
 import clip
 
-class CLIPTextEncoder(Executor):
+class ClipTextEncoder(Executor):
     """Encode text into embeddings."""
 
-    def __init__(self, model_name: str = 'ViT-B/32', *args, **kwargs):
+    def __init__(self,
+                 model_name: str = 'ViT-B/32',
```

Comment:
a lot of parameters are missing, check `CLIPImageEncoder`

Reply:
batch size and device added